lib/src/action.dart

```diff
@@ -20,11 +20,30 @@ import 'package:w_common/disposable.dart';
 
 import 'package:w_flux/src/constants.dart' show v3Deprecation;
 
+/// Like [Action1], but without an expected payload (aka a void payload).
+class Action0 extends Action1<void> {
+  @override
+  String get disposableTypeName => 'Action1';
+
+  @override
+  Future call([void payload]) => super.call(null);
+}
+
+/// Like [Action1], but payloads cannot be made non-nullable since the argument
+/// to [call] is optional.
+class Action<T> extends Action1<T> {
```

Comment:
My thought is that we'd either:
- keep this type around as a common base type, but could change it to an interface
- use Action1 as the common base type where needed